### ISSUE TITLE
Fix CLI view after printing chat

### ISSUE THREAD

**Issue by Noirdemort** [Noirdemort/lacr#1]
After receiving and printing text, prompting arrows don't show up until another text is sent.

**Comment by divyansshhh** [comment]
Can I work on this?

**Comment by Noirdemort** [comment]
Yeah Sure, To test it out on a single machine comment out key generation after running it for one time.

**Comment by divc99** [comment]
I'm having an issue with setup.sh. I changed it to an executable but I'm getting the following error -
`/setup.sh: line 12: syntax error near unexpected token `('
./setup.sh: line 12: `        Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))'`

**Comment by divyansshhh** [comment]
I'm having an issue with setup.sh. I changed it to an executable but I'm getting the following error -
`./setup.sh: line 12: syntax error near unexpected token ('`
`./setup.sh: line 12: Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))'`

**Comment by Noirdemort** [comment]
I haven’t tested script on windows.
Just install OpenSSL and add it to path and you’ll be on your way.

**Comment by Noirdemort** [comment]
For now, I have removed those statements, I'll look into it and fix them later.
